juno/assetlist.json

```diff
@@ -2048,6 +2048,33 @@
         }
       ],
       "coingecko_id": "backbone-labs-staked-juno"
+    },
+	{
+      "description": "Airdrop For All [AFA - New Name on Cosmos Ecosystem, A4A - Old Name on TurtleNetwork] is a token from turtleNetwork towards cosmos ecosystem.",
+      "type_asset": "cw20",
+      "address": "juno1spjes0smg5yp40dl7gqyw0h8rn03tnmve06dd2m5acwgh6tlx86swha3xg",
+      "denom_units": [
+        {
+          "denom": "cw20:juno1spjes0smg5yp40dl7gqyw0h8rn03tnmve06dd2m5acwgh6tlx86swha3xg",
+          "exponent": 0
+        },
+        {
+          "denom": "afa",
+          "exponent": 0
```

Comment:
We cannot have multiple denoms with the same exponent (but we can have "afa" be an alias of the base denom)